[Internal] CI failing for M2 + PHP 7.4 due to composer update and laminas-mail dropping support

<!--- Provide a general summary of the issue in the Title above -->

`composer update` instructions in the "Integration" workflow upgrades laminas-mail from 2.16.0 to 2.17.0 [which dropped support for PHP 7.3 and PHP 7.4](https://github.com/laminas/laminas-mail/releases/tag/2.17.0).

```
./composer.json has been updated
Using version ^0.0.3 for magento/sample-data-media-venia
./composer.json has been updated
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
In Laminas\DependencyPlugin\DependencyRewriterV2::onPrePoolCreate
Updating dependencies
Lock file operations: 3 installs, 7 updates, 0 removals
  - Upgrading laminas/laminas-mail (2.16.0 => 2.17.0)
  - Locking magento/module-catalog-sample-data-venia (0.0.3)
  - Locking magento/module-configurable-sample-data-venia (0.0.3)
  - Locking magento/sample-data-media-venia (0.0.3)
  - Upgrading psr/cache (1.0.1 => 3.0.0)
  - Downgrading spomky-labs/aes-key-wrap (v6.0.0 => v5.0.0)
  - Upgrading symfony/deprecation-contracts (v2.5.2 => v3.1.1)
  - Upgrading symfony/property-info (v5.4.11 => v6.1.3)
  - Downgrading symfony/yaml (v5.3.14 => v5.0.11)
  - Upgrading thecodingmachine/safe (v1.3.3 => v2.2.3)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 3 installs, 7 updates, 0 removals
  - Downloading psr/cache (3.0.0)
  - Downloading spomky-labs/aes-key-wrap (v5.0.0)
  - Downloading laminas/laminas-mail (2.17.0)
  - Upgrading symfony/deprecation-contracts (v2.5.2 => v3.1.1): Extracting archive
  - Upgrading psr/cache (1.0.1 => 3.0.0): Extracting archive
  - Downgrading spomky-labs/aes-key-wrap (v6.0.0 => v5.0.0): Extracting archive
  - Upgrading laminas/laminas-mail (2.16.0 => 2.17.0): Extracting archive
  - Upgrading symfony/property-info (v5.4.11 => v6.1.3): Extracting archive
  - Upgrading thecodingmachine/safe (v1.3.3 => v2.2.3): Extracting archive
  - Downgrading symfony/yaml (v5.3.14 => v5.0.11): Extracting archive
  - Installing magento/module-catalog-sample-data-venia (0.0.3): Extracting archive
  - Installing magento/module-configurable-sample-data-venia (0.0.3): Extracting archive
  - Installing magento/sample-data-media-venia (0.0.3): Extracting archive
Package symfony/debug is abandoned, you should avoid using it. Use symfony/error-handler instead.
[...]
```

The "[Test] Data : Install" step which runs a setup:upgrade then fails because of a PHP 8 only syntax
```
Module 'Magento_CatalogSampleDataVenia':
PHP Fatal error:  Cannot use ::class with dynamic class name in /var/www/magento/vendor/laminas/laminas-mail/src/Message.php on line 394
{"messages":{"error":[{"code":500,"message":"Server internal error. See details in report api\/1393295566889"}]}}
Error: Process completed with exit code 255.
```

### Preconditions
<!--- Please Provide detailed informations about the environment you use -->

<!-- Magento Version : Are you using Magento CE or EE ? Which version of Magento are you using exactly ? -->
Magento Version : 2.4.3 (CE, EE), 2.4.3-p1 (CE, EE)

<!-- ElasticSuite Version : Which exact version of ElasticSuite are you using ? -->
ElasticSuite Version : 2.10.x-dev

--- a/.github/workflows/20-integration.yml
+++ b/.github/workflows/20-integration.yml
@@ -53,7 +53,7 @@ jobs:
                   - php-versions: '8.1'
                     magento-versions: '2.4.5'
                     magento-editions: 'enterprise'
-                    experimental: true                    
+                    experimental: true
 
         continue-on-error: ${{ matrix.experimental }}
 
@@ -206,7 +206,7 @@ jobs:
                 composer remove magento/module-configurable-sample-data-venia --no-update --no-interaction
                 composer remove magento/sample-data-media-venia --no-update --no-interaction
                 composer remove smile/elasticsuite --no-update --no-interaction
-                composer update --no-interaction --ignore-platform-reqs
+                composer install --no-interaction --ignore-platform-reqs
                 composer config discard-changes false
 
             - name: "[Init] Install proper version of Magento through Composer"
@@ -287,7 +287,7 @@ jobs:
                   composer require magento/module-catalog-sample-data-venia:dev-master@dev --no-update --no-interaction --ignore-platform-reqs
                   composer require magento/module-configurable-sample-data-venia:dev-master@dev --no-update --no-interaction --ignore-platform-reqs
                   composer require magento/sample-data-media-venia:dev-master@dev --no-update --no-interaction --ignore-platform-reqs
-                  composer update --no-interaction --ignore-platform-reqs
+                  composer install --no-interaction --ignore-platform-reqs
                 fi
 
             - name: "[Test] Data : Require for Magento >=2.4.2"
@@ -301,7 +301,7 @@ jobs:
                   composer require magento/module-catalog-sample-data-venia --no-update --no-interaction --ignore-platform-reqs
                   composer require magento/module-configurable-sample-data-venia --no-update --no-interaction --ignore-platform-reqs
                   composer require magento/sample-data-media-venia --no-update --no-interaction --ignore-platform-reqs
-                  composer update --no-interaction --ignore-platform-reqs
+                  composer install --no-interaction --ignore-platform-reqs
                 fi
 
             - name: "[Init] Data : Prepare for Magento >= 2.4.4"

